tensorflow/python/keras/losses.py

```diff
@@ -836,7 +836,7 @@ def hinge(y_true, y_pred):
 
   Args:
     y_true: The ground truth values. `y_true` values are expected to be -1 or 1.
```

Comment:
be converted

Reply:
If binary (0 or 1) labels are provided they will be converted to -1 or 1

Reply:
This is already in the docs.

Reply:
I meant in the right hand side in the line you have changed, i did not want to create another comment for this so commented on the prev one.